code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -348,6 +348,11 @@ datum/status_effect/stabilized/blue/on_remove()
 	colour = "metal"
 	var/cooldown = 30
 	var/max_cooldown = 30
+	var/list/blacklisted_stacks = list(/obj/item/stack/sheet/runed_metal)
```

Comment:
remove the variable here and instead make it a direct istype check.

Reply:
seems more maintainable this way, if someone wanted to blacklist another type in the future

Reply:
That's the idea. I am sure there will be snowflake sheets in the future.

Reply:
That is true but it will cost more memory to do this a more cost efficent would be having it as a static typecacheof.